string new_state
---
bool success
string message

